src/components/clothingGrid/ClothingCard.tsx

```diff
@@ -0,0 +1,30 @@
+import TitleWrapper from "./TitleWrapper";
+import DescriptionWrapper from "./DescriptionWrapper";
+import PriceAndLike from "./PriceAndLike";
+
+type CardProps = {
+  title: string;
+  description: string;
+  price: number;
+  image: string;
+};
+
+export default function ClothingCard({
+  title,
+  description,
+  price,
+  image,
+}: CardProps) {
+  return (
+    <div className="flex flex-col gap-4 items-center px-2 py-7 ">
+      <div>
+        <img src={image} />
+      </div>
+      <div className="flex flex-col gap-4 text-center px-12">
+        <TitleWrapper>{title}</TitleWrapper>
+        <DescriptionWrapper>{description}</DescriptionWrapper>
+      </div>
+      <PriceAndLike price={price} />
```

Comment:
sometimes when we only use one prop, and it is a display one, its better to honor HTML standards and set it as a child instead of a prop. Moreover by seeing this component i don't think its worth the extra work of outsourcing it (since its not being used elsewhere). Just render it here:
```suggestion
      <div className="w-full flex justify-between px-16">
      <p className="font-manrope font-normal text-primary-600">€ {price}</p>
      <button>
        <HeartIcon pathClassName="fill-complementary-600" />
      </button>
    </div>
```